src/PluginCleanupInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\ultimate_cron\PluginCleanupInterface.
+ */
+
+namespace Drupal\ultimate_cron;
+
+interface PluginCleanupInterface {
+
+  /**
+   * Inform Ultimate Cron about cron jobs.
```

Comment:
Documentation has been copied from hook_cronapi(), this needs to explain what this is about. Something like "Cleans and purges data stored by this plugin. The class should also have a docblock, something like "Plugins can implement this to be invoked for regular cleanup tasks."